python/guide/packages/gluon/text/index.rst

```diff
@@ -17,6 +17,96 @@ These tutorials will help you learn how to create and use models that work with
 
       How to use a pretrained Transformer model and how to train one that translates between English and German.
 
+Text Tutorials
+==============
+
+These tutorials will help you learn how to create and use models that work with text and other natural language processing tasks.
+          
+.. container:: cards
+          
+   .. card::
+      :title: Neural Machine Translation with GluonNLP
+      :link: gnmt.html
+             
+      How to create a model that translates between English and Vietnamese.
+
+   .. card::
+      :title: Transformer Models with GluonNLP
+      :link: transformer.html
+    
+      How to use a pretrained Transformer model and how to train one that translates between English and German.
+Text Tutorials
```

Comment:
I'd put a line break here for readability and also Sphinx really likes line breaks.